src/Form/selectors-css.js

```diff
@@ -0,0 +1,209 @@
+input[autocomplete="cc-csc"],
+input[autocomplete="csc"],
+input[autocomplete="cc-cvc"],
+input[autocomplete="cvc"],
+input[name="cvc"],
+input[name="cc-cvc"],
+input[name="cc-csc"],
+input[name="csc"],
+input[name="securityCode"]`
+
+const expirationMonth = `
+[autocomplete="cc-exp-month"],
+[name="ccmonth"],
+[name="ppw-expirationDate_month"],
+[name=cardExpiryMonth],
+[name="expiration-month"],
+[name*=ExpDate_Month i],
+[id*=expiration-month i]`
+
+const expirationYear = `
+[autocomplete="cc-exp-year"],
+[name="ccyear"],
+[name="ppw-expirationDate_year"],
+[name=cardExpiryYear],
+[name="expiration-year"],
+[name*=ExpDate_Year i],
+[id*=expiration-year i]`
+
+const expiration = `
+[autocomplete="cc-exp"],
+[name="cc-exp"],
+[name="exp-date"],
+[name="expirationDate"],
+input[id*=expiration i]`
+
+const firstName = `
+[name*=fname i], [autocomplete*=given-name i],
+[name*=firstname i], [autocomplete*=firstname i],
+[name*=first-name i], [autocomplete*=first-name i],
+[name*=first_name i], [autocomplete*=first_name i],
+[name*=givenname i], [autocomplete*=givenname i],
+[name*=given-name i],
+[name*=given_name i], [autocomplete*=given_name i],
+[name*=forename i], [autocomplete*=forename i]`
+
+const middleName = `
+[name*=mname i], [autocomplete*=additional-name i],
+[name*=middlename i], [autocomplete*=middlename i],
+[name*=middle-name i], [autocomplete*=middle-name i],
+[name*=middle_name i], [autocomplete*=middle_name i],
+[name*=additionalname i], [autocomplete*=additionalname i],
+[name*=additional-name i],
+[name*=additional_name i], [autocomplete*=additional_name i]`
+
+const lastName = `
+[name=lname], [autocomplete*=family-name i],
+[name*=lastname i], [autocomplete*=lastname i],
+[name*=last-name i], [autocomplete*=last-name i],
+[name*=last_name i], [autocomplete*=last_name i],
+[name*=familyname i], [autocomplete*=familyname i],
+[name*=family-name i],
+[name*=family_name i], [autocomplete*=family_name i],
+[name*=surname i], [autocomplete*=surname i]`
+
+const fullName = `
+[name=name], [autocomplete=name],
+[name*=fullname i], [autocomplete*=fullname i],
+[name*=full-name i], [autocomplete*=full-name i],
+[name*=full_name i], [autocomplete*=full_name i],
+[name*=your-name i], [autocomplete*=your-name i]`
+
+const phone = `
+[name*=phone i], [name*=mobile i], [autocomplete=tel]`
+
+const addressStreet1 = `
+[name=address], [autocomplete=street-address], [autocomplete=address-line1],
+[name=street],
+[name=ppw-line1]`
+
+const addressStreet2 = `
+[name=address], [autocomplete=address-line2],
+[name=ppw-line2]`
+
+const addressCity = `
+[name=city], [autocomplete=address-level2],
+[name=ppw-city]`
+
+const addressProvince = `
+[name=province], [name=state], [autocomplete=address-level1]`
+
+const addressPostalCode = `
+[name=zip], [name=zip2], [name=postal], [autocomplete=postal-code], [autocomplete=zip-code],
+[name*=postalCode i], [name*=zipcode i]`
+
+const addressCountryCode = `
+[name=country] [autocomplete=country],
```

Comment:
Unrelated to your work, but I just noticed a typo here (missing comma).
```suggestion
[name=country], [autocomplete=country],
```

Reply:
@GioSensation please see this commit where I've changed the CSS selectors https://github.com/duckduckgo/duckduckgo-autofill/commit/18d3a9e5e92a160b0e701f8aa8fea53f9a4a34a9

Reply:
^ could you verify if you think this looks correct